components/Nav.js

```diff
@@ -49,15 +49,16 @@ const navItems = [
     key: 'my-profile',
     title: ({ user }) => {
       if (user) {
+        const { username } = user.attributes
+
         return (
           <React.Fragment>
-            <img
-              alt="User Avatar"
-              className="avatar"
-              height={20}
-              src={`//api.adorable.io/avatars/20/${user.id}`}
-              width={20} />
-            <span>{user.attributes.username}</span>
+            <div
+              aria-label={`${username}'s Avatar`}
+              className="avatar tiny"
+              role="img"
+              style={{ backgroundImage: `url(//api.adorable.io/avatars/20/${user.id})` }} />
```

Comment:
This should be a conditional depending on the state of `user.attributes.avatar`. if `avatar === true`, point to `/api/users/${user.id}/avatar`, else point to `//api.adorable.io/avatars/20/${user.id}`

Reply:
Makes sense.